addons/point_of_sale/static/src/js/db.js

```diff
@@ -481,10 +481,58 @@ var PosDB = core.Class.extend({
         var saved = this.load('unpaid_orders',[]);
         var orders = [];
         for (var i = 0; i < saved.length; i++) {
-            orders.push(saved[i].data);
+            orders.push(saved[i]['data']);
         }
         return orders;
     },
+    /**
```

Comment:
All functions added in this file are only used in pos_restaurant. It can be moved to this module